src/main/java/seedu/address/model/exercise/Exercise.java

```diff
@@ -1,19 +1,51 @@
 package seedu.address.model.exercise;
 
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
 import seedu.address.model.UniqueElement;
+import seedu.address.model.attempt.Attempt;
+import seedu.address.model.attempt.exceptions.MaximumAttemptsReachedException;
+import seedu.address.model.participation.Participation;
 
 /**
  * Represents an Exercise category in a {@link seedu.address.model.competition.Competition}
  */
 public class Exercise extends UniqueElement {
-    private final Type type;
+    private final Lift lift;
+    private ObservableList<Attempt> threeAttempts = FXCollections.observableArrayList();
+
+    public Exercise(Lift lift) {
+        this.lift = lift;
+    }
 
-    public Exercise(Type type) {
-        this.type = type;
+    public Lift getLift() {
+        return lift;
     }
 
-    public Type getType() {
-        return type;
+    /**
+     * Creates a new attempt for an athlete's lift.
+     * @param athlete the participation object that references an athletes participation in the competition.
+     * @param weight weight of the attempt the athlete is going for
+     */
+    public void newAttempt(Participation athlete, int weight) throws MaximumAttemptsReachedException {
+        int newAttemptNo = threeAttempts.size() + 1;
+
+        if (newAttemptNo <= 3) {
```

Comment:
should refrain from using magical numbers.
Consider declaring it as a constant instead like MAXIMUM_ATTEMPTS = 3.